src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx

```diff
@@ -36,17 +37,15 @@ export function getReleaseEventView(
   const {projects, environments, datetime} = selection;
   const {start, end, period} = datetime;
   const releaseFilter = currentOnly ? `release:${version}` : '';
-
-  const toOther = `to_other(release,${version},others,current)`;
-  // this orderby ensures that the order is [others, current]
-  const toOtherAlias = `to_other_release_${version}_others_current`;
+  const toOther = `to_other(release,"${escapeDoubleQuotes(version)}",others,current)`;
```

Comment:
Make sure to quote and escape the version string so we work with special characters.